frontend/src/App.js

```diff
@@ -20,7 +20,7 @@ import { Page } from './Page'
 
 const PageNotFound = lazy(() => import('./PageNotFound'))
 const CreateUserPage = lazy(() => import('./CreateUserPage'))
-const QRcodePage = lazy(() => import('./QRcodePage'))
+// const QRcodePage = lazy(() => import('./QRcodePage'))
```

Comment:
This can probably just be straight up removed, I don't believe we are going to use the QR code again.